src/components/views/CertificateForm.tsx

```diff
@@ -46,54 +113,29 @@ const NewCertificate: React.FC = () => {
 
   const handleResetFields = () => {
     setCertificate({
-      supplier: '',
+      supplier: { name: '' },
       certificateType: '',
       validTo: '',
       validFrom: '',
       preview: undefined,
```

Comment:
preview is confusing for object prop name. This is "pdfUrl" not "preview". Preview would be something else